<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lamp Project Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="taskBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>261</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Tasks</string>
    </property>
    <widget class="QTreeView" name="tree_tasks">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>431</height>
      </rect>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>239</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_add_folder">
        <property name="text">
         <string>New folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add_task">
        <property name="text">
         <string>New task</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_del_task">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Output folder</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="workfilesBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>541</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Work files</string>
    </property>
    <widget class="QPushButton" name="btn_add_workfile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>New from preset</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tbl_workfiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>521</width>
       <height>431</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Software</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_add_existing_workfile">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add existing</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>301</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Projects</string>
    </property>
    <widget class="QListView" name="list_projects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>281</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dailies">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dailies folder</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>158</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_open_project">
        <property name="text">
         <string>Open project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_new_project">
        <property name="text">
         <string>New Project</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="menu_settings"/>
    <addaction name="menu_exit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="menu_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
